TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -189,6 +189,11 @@ codeElement:
 	returnStatement |
 	returnStatementEnd |
 	sortStatement |
+
+//	 [COBOLV6]
+	// --- Dynamic allocation statements ---
+	freeStatement |
```

Comment:
I'll rather a generic name here, like `cobol2002Statement`.
Then redefines it in Cobol2002CodeElements.g4 to include the `freeStatement`

Reply:
See ab10d10110b03c9420497858c93592d7498b6a42.